### DISCUSSION TITLE
switch or change tabs (made by neo-tree)

### DISCUSSION THREAD

**Discussion by jellis206** [LazyVim/LazyVim#156] [Q&A] [answered]
I can make new tabs and switch tabs using the default key maps however when I try to close them they don't actually close. Also trying to jump around to different tabs created by neo-tree does not work. I am curious if anyone has had the same issue / their solution. I am pretty new to using neovim so maybe this is more of a user error.

**Comment by luxus** [comment]
i guess a better place would be [neo-tree](https://github.com/nvim-neo-tree/neo-tree.nvim)

have you tried "?" in neo-tree it shows you all mappings

**Reply by jellis206** [reply]
I will give that a try

**Comment by folke** [comment, marked as answer]
When you create a new tab, you also create a new buffer. When you then close that new tab, the new buffer is stil there. You need to close that too. So the tabs do close. You just don't fully seem to understand the difference between tabs and buffers

**Reply by jellis206** [reply]
Ahhh that would make a lot of sense, thank you for the clarification!
